examples/express/index.js

```diff
@@ -5,15 +5,16 @@ const app = express();
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'; //just for dev
 
 app.get("/tofu", (request, response) => {
-  const jwtVerifier = new PomeriumVerifier({});
-  jwtVerifier.verifyJwt(request.get('X-Pomerium-Jwt-Assertion')).then(r => response.send(r))
+  const jwtVerifier = new PomeriumVerifier({expirationBuffer: 1000});
```

Comment:
can we dig in deeper to the reasons we're observing timing issues on a local machine ?

Reply:
In a later pr